xcodebuild/xcodebuild.go

```diff
@@ -158,22 +158,19 @@ func createXcodebuildOptions(xctestrun string, onlyTesting, skipTesting []string
 	}
 
 	if 0 < len(onlyTesting) {
-		// This is a string on purpose. It was an array before and the individual parameter escaping (putting it in quotes like "-only-testing:test-identifier")
-		// was not working as xcodebuild did not recognize the arguments. But if all of these parameters are put in a single string and escaped once, it works.
-		var args string
```

Comment:
I reverted this because it turns out Xcode will only use the first 1000 config value from a long list. And most probably the empty lines caused an issue in the previous implementation.